### PR TITLE
Support COVERALLS_PARALLEL environment variable

### PR DESCRIPTION
**Pull request by vtjnash**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,10 +12,12 @@ julia:
   - nightly
 
 notifications:
-    email: false
+  webhooks: https://coveralls.io/webhook
+  email: false
 
 script:
   - julia --check-bounds=yes etc/travis-test.jl
+  - export COVERALLS_PARALLEL=true
```
Comment (fingolfin) [inline .travis.yml]: Not a reason to hold up this PR, just wondering, but: is there any reason to do it like this, and now how the README suggests to do i? I.e.,
```
env:
  global:
    - COVERALLS_PARALLEL=true
```

Comment (vtjnash) [reply .travis.yml]: Just that it confuses testing to set it globally. We similarly don't use the suggested script either.

**Comment by vtjnash** [comment]
> I'll be happy to review this again once it has stabilized.

Yeah, was just making all of the edits in the GitHub editor, so I could only do one file at a time. If it wasn't clear, it should be done now. I planned to squash-and-merge.

File: README.md [hunk 2]
```diff
@@ -173,6 +173,18 @@ When using Coverage.jl locally, over time a lot of `.cov` files can accumulate.
...
+    For Travis, this can be achieved by adding the following to `.travis.yml`:
+
+        env:
+          global:
+            - COVERALLS_PARALLEL=true
+        notifications:
+          webhooks: https://coveralls.io/webhook
+
+
+
```
Comment (fingolfin) [inline README.md]:
```suggestion
```

**Pull request merged**